### DISCUSSION TITLE
Bug: MacOS-to-linux SSH Option+<action> visual glitch

### DISCUSSION THREAD

**Discussion by kaspar-p** [ghostty-org/ghostty#4105] [Q&A] [answered]
On build 8469, universal MacOS Binary.

For example:
 - Option+Left should go left by a word
 - Option+Right should go right by a word
 - Option+Backspace should delete a word

This works fine on the MacOS side, but as soon as I SSH into a linux host, it breaks.

Typing a command:
![Screenshot 2024-12-30 at 13 09 25](https://github.com/user-attachments/assets/6e03a164-c6e0-449e-89c7-a633e812e5fa)

Pressing Option+Backspace to delete the command, if I made a mistake:
![Screenshot 2024-12-30 at 13 10 17](https://github.com/user-attachments/assets/7c5209fe-ba4f-4065-bb20-83a1c95c2537)

It just sort of "tabs" a bit of spacing. Pressing enter at this point gets me a new terminal line and it doesn't execute the `ls`, but is visually messy and doesn't allow me to iterate on the commands I want to run very easily.

**Comment by mitchellh** [comment, marked as answer]
https://ghostty.org/docs/help/terminfo

**Reply by kaspar-p** [reply]
That works, thanks!
